about.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <script defer src="script.js"></script>
+</head>
+
+<body>
+   
+    <nav class="fixed-nav">
+        <!-- <div class="container"> -->
+            <ul>
+                <li><a href="#">
+                        <h1>F</h1>
+                    </a></li>
+                <!-- <li><a href="#"><img src="images/twitterlogooutline_80724.png"></a></li> -->
+                <li><a href="#">
+                        <h1>English</h1>
+                    </a></li>
+                <li><a href="#">
+                        <h1>My Page</h1>
+                    </a></li>
+                <li><a href="#">
+                        <h1>Logout</h1>
+                    </a></li>
+            </ul>
+        <!-- </div> -->
+    </nav>
+
+
+
+
+    <nav class="nav-bar">
+        <!-- <div class="container"> -->
+            <a href="index.html"><img class="logo" src="images/logo.png" alt="Logo"></a>
+            <div class="nav-menu">
+                <ul class="nav-links">
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Program</a></li>
+                    <li><a href="#">Join</a></li>
+                    <li><a href="#">Sponsor</a></li>
+                    <li><a href="#">News</a></li>
+                    <button class="CC">CC Campain</button>
+                </ul>
+            </div>
+
+            <div class="humburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+        <!-- </div> -->
+    </nav>
+    
+
```

Comment:
- [ ] Lastly, please effect the above spacing error in your mobile version also 😊